odk1-src/launch-apps-from-collect.rst

```diff
@@ -21,7 +21,21 @@ Launching External Apps
 Launching external apps to populate single fields
 ------------------------------------------------------
 
-ODK Collect can launch external applications to populate string, integer or decimal fields using the ``ex:intentString`` appearance. A ``value`` parameter that holds the current value for that field is passed to the application. Since v1.4.3, additional parameters can be specified. The names of these parameters are user defined and ``value`` is the only reserved name. 
+ODK Collect can launch external applications to populate string, integer or decimal fields using the ``ex:intentAction(intentExtras)`` appearance.
+
+Under the covers, this feature configures an intent, a messaging objects used to request an action from another app component. For more details on intents,  refer to `these Android docs <https://developer.android.com/guide/components/intents-filters.html>`_.
```

Comment:
> an intent, a messaging objects used to request an action

I *think* this is supposed to be `object` not `objects`.

But I think even then, the multiple commas make it a little unclear.
Maybe something like...

```
Under the hood, this feature configures an `intent`_. An intent is a messaging object used to...

.. _intent: https://developer.android.com/guide/components/intents-filters.html
```
(And then remove  "For more details...")

(Could put the link on the explanatory sentence instead of the word "intent" if that feels better.)